course/lesson-11-time-delta/rotating-using-delta/TestsRotatingDelta.gd

```diff
@@ -22,20 +22,89 @@ func _prepare() -> void:
 
 func test_rotating_character_is_time_dependent() -> String:
 	if not _has_proper_body:
+		#reverse find for any delta at all
 		var has_delta: bool = _body.rfind("delta") > 0
-		var is_in_parentheses: bool = _body.rfind(")")
+		#set parentheses found false by default
+		var is_in_parentheses: bool = false
 
+		#Regular expression to check if delta is inside parenthesis
+		#I know, regexp is ancient evil magic, as the quote goes:
```

Comment:
Regexes are perfectly fine! Next time could you remove the comments? We can read regexes and the variable name already documents what it does